### DISCUSSION TITLE
Is there a gh cli command to list self-hosted runners for a repo?

### DISCUSSION THREAD

**Discussion by pantelis-karamolegkos** [community/community#54228] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

(that's it, this is the question)

**Comment by daleobrien** [comment, marked as answer]
This works for me: ```gh api  -H "Accept: application/vnd.github+json"  -H "X-GitHub-Api-Version: 2022-11-28"  /orgs/YOUR-ORG-SLUG/actions/runners?per_page=100```
